### PR TITLE
Avoid recomputing FGD tables every frame

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
### Purpose of this PR
Check if FGD tables for AxF have been already init, if so don't recompute them again. 
---
### Release Notes
Avoid preintegration of AxF's FGD tables every frame. 

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=AxF-fix-preintegrated-FGD-every-frame&automation-tools_branch=add-platform-filter&unity_branch=trunk [Still running]

---
### Overall Product Risks
**Technical Risk**:  Low

**Halo Effect**: Low

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.cs [hunk 1]
```diff
@@ -196,6 +198,8 @@ public override void RenderInit(CommandBuffer cmd)
                 CoreUtils.DrawFullScreen(cmd, m_preIntegratedFGDMaterial_Ward, new RenderTargetIdentifier(m_preIntegratedFGD_Ward));
                 CoreUtils.DrawFullScreen(cmd, m_preIntegratedFGDMaterial_CookTorrance, new RenderTargetIdentifier(m_preIntegratedFGD_CookTorrance));
             }
+
+            m_precomputedFGDTablesAreInit = true;
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.cs]: you miss to reset it to false 
m_precomputedFGDTablesAreInit = false 
in Cleanup() function

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.cs]: Note: HDRP is frequently rebuild (i.e init/cleanup cycle happen often)

Comment (FrancescoC-unity) [reply com.unity.render-pipelines.high-definition/Runtime/Material/AxF/AxF.cs]: Ha, yes,  I ignored the existence of the Cleanup function, I just hastily did the change :-) Will fix immediately!

**Comment by sebastienlagarde** [comment]
Note: katana is green except the test from LW that was failing on master so we are good. Merging: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=AxF-fix-preintegrated-FGD-every-frame&automation-tools_branch=add-platform-filter&unity_branch=trunk#

**Pull request merged**